onchain/src/social/interface.cairo

```diff
@@ -0,0 +1,39 @@
+// Generate interface using src5.rs
+// -> Copy interface into a file and run `src5_rs parse <PATH_TO_CAIRO_FILE>`
+// -> Note: You would need to define custom Structs and Enums in the file. 
+pub const ISRC5_ID: felt252 = 0x3983c447d2357bbd8e577096a70949166b5337acbb2b66f279157b8f352ff8f;
+
+// Below is the interface structure for generating the interface ID
+// You can generate the ID using this tool: https://github.com/ericnordelo/src5-rs/tree/main?tab=readme-ov-file
+struct Transfer {
```

Comment:
This is because it's in the original interface in account.cairo.
This file contains the interface for computing the ID so it needs to be the same as the original interface.

Reply:
I think I should make a video explaining why. I will send the video explanation soon.

Reply:
@maciejka, Here the link to the recorded [video explanation](https://www.loom.com/share/5d4767f69dff4569b371e8adbc3ef6be?sid=c2ca43e9-8273-4441-a7ac-fe4b5cd8aea7)

Reply:
Ok, I see you point. 
It looks like it is a problem with src5_rs, it can't parse cairo code properly.

In this case let's move `ISocialAccount` back to the account.cairo. We will worry about src5 id generation later.

Reply:
Ok, I see you point.
It looks like it is a problem with src5_rs, it can't parse cairo code properly.

In this case let's move ISocialAccount back to the account.cairo. We will worry about src5 id generation later.